emissionsapi/web.py

```diff
@@ -82,11 +82,18 @@ def get_data(session, country=None, geoframe=None, polygon=None,
         except RESTParamError as err:
             return str(err), 400
 
+    # parse paramter point
+    if point is not None:
```

Comment:
```suggestion
    elif point is not None:
```
…since it doesn't make sense to continue with e.g. both point and polygon, does it?